Update mapping in csv-import-mapping.json

souravcipher opened the pull request:
This PR includes the latest mapping

souravcipher commented on env.js:
```diff
@@ -18,8 +18,8 @@ debug.enable(process.env.DEBUG);
 
 const defaults = {
   NODE_ENV: 'development',
-  API_KEY: '09u624Pc9F47zoGLlkg1TBSbOl2ydSAq',
```
This is an old key and already deleted!

znarf replied:
You should not touch that file.

znarf commented on env.js:
```diff
@@ -18,8 +18,8 @@ debug.enable(process.env.DEBUG);
 const defaults = {
   NODE_ENV: 'development',
-  API_KEY: '09u624Pc9F47zoGLlkg1TBSbOl2ydSAq',
-  API_URL: 'https://api-staging.opencollective.com',
```
You should not touch that file.

znarf commented:
I updated the instructions at https://github.com/opencollective/opencollective-tools#prerequisite

`PERSONAL_TOKEN` (renamed from API_KEY) and `API_URL` should be put in a file named `.env`

`env.js` should never be touched.

souravcipher commented:
> I updated the instructions at https://github.com/opencollective/opencollective-tools#prerequisite
> 
> `PERSONAL_TOKEN` (renamed from API_KEY) and `API_URL` should be put in a file named `.env`
> 
> `env.js` should never be touched.

Thank you so much @znarf, I have updated the PR!!
